common/constants.js

```diff
@@ -239,6 +239,11 @@ module.exports = Object.freeze({
     HOSTNAME: 'servicefabrik.io',
     NAMESPACE: 'default',
     API_VERSION: 'v1alpha1',
+    ENCRYPTION: {
```

Comment:
Would prefer if it is moved out of the APISERVER object since it may be required later for some other operation

Reply:
Done!